bsconfig.json

```diff
@@ -16,7 +17,7 @@
   ],
   "package-specs": [
     {
-      "module": "commonjs",
+      "module": "es6",
```

Comment:
This appears to be the reason for the build failure -> Jest doesn't like ES6 `import` syntax (a tale as old as time).